wiki/Beatmap_Spotlights/Seasons/2021_Spring/id.md

```diff
@@ -26,17 +25,17 @@ no_native_review: true
 
 ## Lobi multiplayer mingguan
 
-Untuk saat ini, bergabung dengan lobi multiplayer mingguan mewajibkan peserta untuk mengunduh dan menginstal [osu!(lazer)](https://github.com/ppy/osu/releases).
+Untuk saat ini, bergabung dengan lobi multiplayer mingguan mengharuskan peserta untuk mengunduh dan menginstal [osu!(lazer)](https://github.com/ppy/osu/releases).
 
-1. Setiap Jumat pada satu musim, satu timeshift multiplayer lobby dibuka. Semua orang dapat berpartisipasi selama minggu tersebut.
+1. Setiap minggu pada satu musim, satu timeshift multiplayer lobby dibuka. Semua orang dapat berpartisipasi selama minggu tersebut.
```

Comment:
It doesn't say that it's every Friday in the original English article

Reply:
still unsure about the previous, Jumat may be used to indicate the day instead a week.

Reply:
I will mark this as resolved. this change is for content parity